sending/src/main/java/org/kin/sending/presenter/RecipientAddressPresenterImpl.java

```diff
@@ -62,9 +94,19 @@ public void setRecipientAddress(@NonNull String address) {
         sendKinPresenter.setRecipientAddress(recipientAddress);
     }
 
+    @Override
+    public void onResume() {
```

Comment:
@yosriz I think he can load it once in onViewCreated or presenter onAttach or something like that, what do you think?

Reply:
@drEfrati when priniting out logs on resume and on attach they are both called - so no difference